src/plugins/irc-events/link.js

```diff
@@ -129,10 +134,16 @@ function fetch(url, cb) {
 			} catch(e) {
 				type = {};
 			}
+			try {
+				size = req.response.headers['content-length'];
+			} catch(e) {
+				size = {};
+			}
```

Comment:
Maybe something more explicit than just catching all exceptions ?(I know, some existing code does like yo did), + are you sure `{}` is a valid size ? ;-)